druid/src/widget/button.rs

```diff
@@ -52,6 +52,27 @@ impl<T: Data> Button<T> {
         }
     }
 
+    /// Create a new button with the provided label.
```

Comment:
I would link to `Label` here.

Reply:
Sure, I'll update the doc comment on `Button::new` as well